split the dictionary by tabs

Bachstelze opened the pull request:
added support for dictionaries with two or more words as translation

mbanon commented:
Hi Bachstelze,
thanks for your contribution. 
Unfortunately, we cannot accept it because this way of treating multiword translations won't work with some features of our Bicleaner classifier (it would require further and deeper changes)
Thanks again.

Bachstelze commented:
Could we change the load method to filter single word translations?
```
    def load(self,fd):
        for line in fd:
            line  = line.rstrip("\n")
            # we only use single word translation with a probability
            if len(line.split()) == 2:
	        parts = line.split()
	        l1    = parts[1]
	        l2    = parts[0]
	        prob  = float(parts[2])
	        if prob <= 0.0000001:
		    continue
	        if not self.alpha.match(l1):
	       	    continue
	        if prob < self.minprob:
		    self.minprob = prob
	        if l1 not in self.d:
		    self.d[l1] = dict()
	        self.d[l1][l2] = prob
```

Or at least specify it in the documentation. Otherwise, it is misleading for dictionaries which aren't generated with GIZA.

mbanon commented:
Hi again,
filtering out multiword translation might be a viable option, 
we will study it and, if it doesn't "break" anything, we can add it for next Bicleaner version.
For now, I'll change the documentation to make it clear than translations must be single word.
Thanks!
